source/components/conversation/FormDecorator.js

```diff
@@ -32,7 +32,6 @@ export var FormDecorator = formType => RenderField =>
 				dispatch(change('conversation', field, value))
 		})
 	)
-	@translate()
```

Comment:
This call is (usually) necessary when we want the component to re-render on switching translations, I'm not sure I understand how this fixes the problem you mentioned.

Reply:
For now, I don't understand how this is working. I've noticed that the translation does not append for the "modifier" link on the last page of the simulation if this @translate() is present in this file. I've checked that translations switching is still working even if it is not there. 

I wonder if this is working due to the fact that FormDecoractor is a sub part of something else already containing a @translate().

I agree that this needs more investigation.